index.html

```diff
@@ -0,0 +1,472 @@
+          <img width="100" src="https://rukminim1.flixcart.com/flap/128/128/image/29327f40e9c4d26b.png?q=100"
+            alt="Grocery" />Grocery
+        </li>
+        <li class="item px-2 d-flex flex-column m-2 align-items-center">
+          <img width="100" src="https://rukminim1.flixcart.com/flap/128/128/image/22fddf3c7da4c4f4.png?q=100"
+            alt="Mobiles" />Mobiles
+        </li>
+        <li class="item px-2 d-flex flex-column m-2 align-items-center">
+          <img width="100" src="https://rukminim1.flixcart.com/flap/128/128/image/c12afc017e6f24cb.png?q=100"
+            alt="Fashion" />
+          <div class="detail-item-name d-flex align-items-center">Fashion
+            <span class="material-symbols-outlined">expand_more</span>
+          </div>
+        </li>
+        <li class="item px-2 d-flex flex-column m-2 align-items-center">
+          <div class="item5-image">
+            <img width="100" src="https://rukminim1.flixcart.com/flap/128/128/image/69c6589653afdb9a.png?q=100"
+              alt="Electronics" />
+          </div>
+          <div class="detail-item-name d-flex align-items-center">Electronics
+            <span class="material-symbols-outlined">expand_more</span>
+          </div>
+        </li>
+        <li class="item px-2 d-flex flex-column m-2 align-items-center">
+          <div class="item6-image">
+            <img width="100" src="https://rukminim1.flixcart.com/flap/128/128/image/ab7e2b022a4587dd.jpg?q=100"
+              alt="Home" />
+          </div>
+          <div class="detail-item-name d-flex align-items-center">Home
+            <span class="material-symbols-outlined">expand_more</span>
+          </div>
+        </li>
+        <li class="item px-2 d-flex flex-column m-2 align-items-center">
+          <div class="item7-image">
+            <img width="100" src="https://rukminim1.flixcart.com/flap/128/128/image/0ff199d1bd27eb98.png?q=100"
+              alt="Appliances" />
+          </div>
+          <div class="detail-item-name d-flex align-items-center">Appliances
+            <span class="material-symbols-outlined">expand_more</span>
+          </div>
+        </li>
+        <li class="item px-2 d-flex flex-column m-2 align-items-center">
+          <div class="item8-image">
+            <img width="100" src="https://rukminim1.flixcart.com/flap/128/128/image/71050627a56b4693.png?q=100"
+              alt="Travel" />
+          </div>
+          <div class="item8-name">Travel</div>
+        </li>
+        <li class="item px-2 d-flex flex-column m-2 align-items-center">
+          <div class="item9-image">
+            <img width="100" src="https://rukminim1.flixcart.com/flap/128/128/image/dff3f7adcf3a90c6.png?q=100"
+              alt="Beauty, Toys & More" />
+          </div>
+          <div class="item9-name">Beauty, Toys & More</div>
+        </li>
+      </ul>
+    </div>
+  </section>
+  <!-- Products header -->
+  <section class="products-header">
```

Comment:
Many classes are used which are not part of `.css` file or `bootstrap`. You should remove them.